tests/publiccloud/azure_aitl.pm

```diff
@@ -0,0 +1,130 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in Azure using azure-cli binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use mmapi 'get_current_job_id';
+use utils qw(zypper_call);
+use JSON;
+use XML::LibXML;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $job_id = get_current_job_id();
+
+    assert_script_run('az version');
+
+    my $provider = $self->provider_factory();
+
+    my $region = get_var('PUBLIC_CLOUD_REGION', 'westeurope');
+    my $resource_group = "openqa-aitl-$job_id";
+    my $subscription_id = $provider->provider_client->subscription;
+
+    my $aitl_client_version = "20241118.1";
```

Comment:
Yes, but what happens when that version is obsolete? Does it even need to be frozen like that, is using "master" too risky?

Reply:
MS are notorious for changing things rapidly and AITL / LISA seems to be very BETA / experimental at this stag.

 This version is actually tagged in the repo: https://github.com/microsoft/lisa/releases so I doubt it will disappear before we update it. e.g. The oldest available tag in the drop-down in GIthub is from `2022` so I think we're good.